source/code-snippets/authentication/aws.js

```diff
@@ -5,12 +5,17 @@ const accessKeyId = encodeURIComponent("<AWS_ACCESS_KEY_ID>");
 const secretAccessKey = encodeURIComponent("<AWS_SECRET_ACCESS_KEY>");
 const clusterUrl = "<MongoDB cluster url>";
 
+const sessionToken = encodeURIComponent("<AWS_SESSION_TOKEN>"); // Optional
```

Comment:
Let's follow the same format as the other code comments for this line and help the reader understand why it's "optional" by explaining in what case they would need it. Since it's optional, maybe it should be commented out and the comment preceding it should also mention that they should uncomment the line if they need it.

Reply:
accepted